### DISCUSSION TITLE
About 3D Multigame.

### DISCUSSION THREAD

**Discussion by mibino** [pokepetter/ursina#678] [General]
I am a developer who is new to Ursina, and I would like to know how to create a simple 3D multiplayer game?

**Comment by ThatBubbledGuy** [comment]
Just join the ursina discord server, it's better there

**Comment by Dinorigami-SLHIDE** [comment]
You can use the networking.py file in the ursina module folder with : `from ursina.networking import*`

Initiate the server from `server.py` : 
`rpc_peer = RPCPeer()
 rpc_peer.start(<ip>, <port>, is_host=True)`

Connect to the server from `client.py` : 
`rpc_peer = RPCPeer()
 rpc_peer.start(<ip>, <port>, is_host=False)`

You can find out more details on the documentation : https://www.ursinaengine.org/networking.html#~~%20Introduction%20to%20Ursina%20Networking%20~~
